pyanaconda/modules/storage/devicetree/handler.py

```diff
@@ -82,16 +82,20 @@ def teardown_device(self, device_name):
             msg = "Failed to tear down {}: {}".format(device_name, str(e))
             raise DeviceSetupError(msg) from None
 
-    def mount_device(self, device_name, mount_point):
+    def mount_device(self, device_name, mount_point, options=""):
```

Comment:
I don't think that it needs to be optional.

Reply:
It was just in case but yeah I don't see any problem with removing this.